src/js/components/IconTextButton.jsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { THEME } from "../constants";
+
+import IconButton from "./IconButton";
+
+const OuterButton = styled.button`
+  align-items: center;
+  background-color: ${({ $background }) => ($background ? "#434343" : "transparent")};
+  box-shadow: 0px 3px 6px #00000029;
+  border-radius: ${({ $invertBorderRadius }) =>
+    $invertBorderRadius ? "6px 6px 0px 0px" : "0px 0px 6px 6px"};
+  cursor: pointer;
+  display: flex;
+  height: 100%;
+  justify-content: space-between;
+`;
+
+const ButtonLabel = styled.span`
+  color: var(--white);
+  font-size: 18px;
+  font-weight: var(--unnamed-font-weight-medium);
+  letter-spacing: 0px;
+  padding: 0 0.5rem;
+  text-align: left;
+  text-decoration: ${({ $active }) => ($active ? "underline" : "none")};
+
+  // TODO: The icon and the label should both have the same hover state when
+  // the outer div is hovered over
+  &:hover {
+    text-decoration: underline;
+  }
+`;
+
+function IconTextButton({
```

Comment:
Can we name it to just `IconButton` since all buttons have text describing what they do.

Reply:
We already have an IconButton component without the associated label (which we will need separately from the IconTextButton component).